data/data-pipeline/data_pipeline/comparison_tool/README.md

```diff
@@ -0,0 +1,35 @@
+# Score comparisons
+
+## Comparison tool
+
+## Single comparator score comparisons
```

Comment:
hmmmm -- wdyt @esfoobar-usds?

Reply:
Let's talk about this in our pair tomorrow!

Reply:
What did y'all decide on this one? Would love to have an easy one-liner to re-run everything. It could be a standalone script, or part of our `application.py`, or something else. 

Otherwise, if I'm new to the codebase, I have to read these scripts to find out how the template notebooks work, then try to compare the template notebooks against the YAMLs in order to figure out which YAMLs are supposed to go with which notebooks, etc.

Reply:
Still pending!

Reply:
we did a simple shell script